crates/rpc/src/lib.rs

```diff
@@ -104,10 +126,22 @@ Hint: If you are looking to run two instances of pathfinder, you must configure
         let module = v02::register_methods(module)?;
         let module = v03::register_methods(module)?;
         let module = pathfinder::register_methods(module)?;
+        let module = match &self.ws_senders {
+            Some(ws_senders) => websocket::register_subscriptions(module, ws_senders.clone())?,
+            None => module,
+        };
+
         let methods = module.build();
 
         Ok(server.start(methods).map(|handle| (handle, local_addr))?)
     }
+
+    pub fn get_ws_senders(&self) -> WebsocketSenders {
+        match &self.ws_senders {
+            Some(txs) => txs.clone(),
+            _ => WebsocketSenders::with_capacity(1),
+        }
+    }
```

Comment:
Why not just clone `self.ws_senders`?

```suggestion
    pub fn get_ws_senders(&self) -> Option<WebsocketSenders> {
        self.ws_senders.clone()
    }
```

Reply:
A bunch of stuff uses `WebsocketSenders`. This allows everything to still work as is, but nothing is actually done unless coz no one can subscribe.
Is this a bad practice?

Reply:
<img width="519" alt="image" src="https://github.com/eqlabs/pathfinder/assets/11048263/30cfd1bf-e679-4c58-81e6-11cb11139e03">